client/download.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/maxmind/geoipupdate/v7/internal/vars"
 )
 
+// HTTPError is an error from performing an HTTP request and receiving a non-200 status code.
+//
+// See https://dev.maxmind.com/geoip/docs/web-services/responses/#errors for more details.
```

Comment:
Although this link is not exactly right as it is for the GeoIP web services, I can see how it would be useful. We unfortunately do not have API docs for the download service. I think we can leave it as is given this.